### ISSUE TITLE
it's all gone

### ISSUE THREAD

**Issue by miko1112** [aouwt/ti-nspire#1]
there's nothing here..........

**Comment by aouwt** [comment]
yeah i was gonna put some stuff here but i haven't found the time to get all the software to transfer the files set up

but i should probably get on that soon
